### ISSUE TITLE
master cannot find available work ,but work is available when make pressure test

### ISSUE THREAD

**Issue by shaolei7788** [Alluxio/alluxio#16848]
**Alluxio Version:**
What version of Alluxio are you using?
2.9.0
**Describe the bug**
master cannot find available work ,but work is available when make pressure test and UI Page Browse cannot open
use command: $ALLUXIO_HOME/bin/alluxio runClass alluxio.stress.cli.StressMasterBench --operation CreateFile  --warmup 5s --duration 3m --create-file-size 1m  --threads 5 --base alluxio:///stress02061811

the master log content is:
2023-02-06 09:19:17,883 WARN  DefaultFileSystemMaster - The persist job (id=1675674699536) for file /stress02061714/files/local-task-0/41843 (id=1419117592575) failed: Task execution failed: Job worker(192.168.0.201) was lost before the task(0) could complete

**To Reproduce**
Steps to reproduce the behavior (as minimally and precisely as possible)

**Expected behavior**

**Urgency**
Describe the impact and urgency of the bug.
high
**Are you planning to fix it**
Please indicate if you are already working on a PR. 
i cannot
**Additional context**

**Comment by LuQQiu** [comment]
@shaolei7788 are the alluxio cluster (master, worker) are function normally?

**Comment by LuQQiu** [comment]
@jja725

**Comment by shaolei7788** [comment]
> 
no,the master log show cannot find available work,but i think the master is not function normally

**Comment by shaolei7788** [comment]
![image](https://user-images.githubusercontent.com/39719096/217126895-0b11354e-1d92-48fd-8bfc-7fdeaf665594.png)
this is master log

<img width="1910" alt="企业微信截图_8a91e80b-f881-42f5-9e68-766ae2103704" src="https://user-images.githubusercontent.com/39719096/217127020-767fd9f0-7449-4504-ac46-2bd3dbd17d4f.png">
this is alluxio-worker log

<img width="1048" alt="企业微信截图_5440a353-d888-4c9f-a96a-6a9b5d8fcf4e" src="https://user-images.githubusercontent.com/39719096/217127111-62849907-6632-4fec-b114-b14be34b1bef.png">
this is alluxio-job-worker log

We have three work and one master

**Comment by shaolei7788** [comment]
<img width="1918" alt="企业微信截图_2d1e121e-6a03-4b34-8ce4-93f441648988" src="https://user-images.githubusercontent.com/39719096/217130656-381ec337-110d-4765-8bb8-b110934a9e83.png">
